app/models/user.rb

```diff
@@ -1,6 +1,4 @@
 class User < ApplicationRecord
-  # Include default devise modules. Others available are:
```

Comment:
- [ ] In `[Blog-App---ROR](https://github.com/denscholar/Blog-App---ROR/tree/add-authorization/feature)/[app](https://github.com/denscholar/Blog-App---ROR/tree/add-authorization/feature/app)/[models](https://github.com/denscholar/Blog-App---ROR/tree/add-authorization/feature/app/models)/ability.rb` please make sure there is no commented code present. 

![image](https://user-images.githubusercontent.com/58504655/180617698-67544434-c0a0-4043-9b01-c2eb2061807e.png)

Reply:
@arslanbisharat Removed these comments. Thanks for the suggestion.